privacyidea/api/lib/prepolicy.py

```diff
@@ -969,7 +969,9 @@ def check_base_action(request=None, action=None, anonymous=False):
     (role, username, realm, adminuser, adminrealm) = determine_logged_in_userparams(g.logged_in_user, params)
 
     # In certain cases we can not resolve the user by the serial!
-    if action not in [ACTION.AUDIT]:
+    if action is ACTION.AUDIT:
```

Comment:
So if i understand this correctly we have to exclude the Audit action here since the user/realm/resolver parameter in the request represent the query for the audit log and not the actual logged in user?
We should explain this somehow for future code reviews :-)

Reply:
So you mean we simply should add a bit more comments in the code?

Reply:
@plettich added more comments